src/ipfs.io/docs/news/index.html

```diff
@@ -0,0 +1,51 @@
+- [Oct 12 - Epicenter Bitcoin Interviews Juan Benet - Decentralizing The Web With The Inter-Planetary File System (IPFS)](https://epicenterbitcoin.com/podcast/100/)
+- [Oct 12 - Decentralizing the Web with IPFS](http://blog.resellerclub.com/decentralizing-the-web-with-ipfs/)
+- [Oct 9 - FreeNAS Alpha ships with IPFS as a transport](http://www.freenas.org/whats-new/2015/10/announcing-freenas-10-alpha.html)
+- [Oct 6-8 - AndyetConf - Richland, WA](http://andyetconf.com)[video (soon)]()
+- [Oct 4 - TechCruch - Why The Internet Needs IPFS Before It’s Too Late](http://techcrunch.com/2015/10/04/why-the-internet-needs-ipfs-before-its-too-late)
+- [Sep 23 - IPFS — Inter-Planetary File system. A for-life, decentralised, privacy respecting web.](https://medium.com/@mvxlr/ipfs-inter-planetary-file-system-65466e4129c6)
+- [Sep 18 - Motherboard, The InterPlanetary File System Wants to Create a Permanent Web](http://motherboard.vice.com/read/the-interplanetary-file-system-wants-to-create-a-permanent-web)
+- [Sep 11 - Container Camp - Containers at Hyperspeed](https://container.camp)[video](https://www.youtube.com/watch?v=vaIWRyotz4g) [slides](https://speakerdeck.com/jbenet/containers-at-hyperspeed)
+- [Sep 10 - First steps toward implementing distributed permanent web - IPFS - Hacked.com](https://hacked.com/first-steps-toward-implementing-distributed-permanent-web-ipfs/)
+- [Sep 8 - HTTP is obsolete. It's time for the distributed, permanent web](https://blog.neocities.org/its-time-for-the-permanent-web.html) [HN](https://news.ycombinator.com/item?id=10187555)
+- [Aug 27 - C-Base IPFS Meetup, Berlin, Germany](https://github.com/ipfs/community/issues/41)
+- [Aug 25 - Software Engineering Daily Interviews Juan Benet](http://softwareengineeringdaily.com/2015/08/25/interplanetary-file-system-ipfs-with-juan-benet/)
+- [Aug 13-17 - Towards Universal Access to All Knowledge: Internet Archive - Chaos Communication Camp, Zehdenick, Germany](https://www.youtube.com/watch?v=lKvoVxUQKD0)
+- [Aug 3-9 - Battlemesh, Maribor, Slovenia](http://battlemesh.org/) [video](https://www.youtube.com/watch?v=OAkJAPS5yoQ)
+- [Jul 23 - Surf Incubator IPFS Meetup, Seattle, WA](http://www.meetup.com/Seattle-IPFS-Meetup/events/224077819/) [video - Kyle Drake Talk](https://vimeo.com/137657331)
+- [Jul 22 - Ctrl IPFS Meetup, Portland, OR](http://attending.io/events/ipfs-portland-meetup-the-permanent-distributed-web)
+- [Jul 22 - Holger Krekels Keynote - EuroPython 2015, Bilbao, Spain](http://dietzel.me/2015/08/02/EuroPython-2015-Holger-Krekels-Keynote-about-the-interplanetary-filesystem-Wed-22nd-July-2015/)
+- [May 23-24 - Data Terra Nemo - Berlin, Germany](http://dtn.is) [video (soon)]()
+- [May 22 - Ethereum IPFS Meetup, Berlin, Germany](https://www.youtube.com/watch?v=QlsBU2moRK4)
+- May 5 - BlockChain University [Part1](https://www.youtube.com/watch?v=JhE_J1-BKJE) [Part2](https://www.youtube.com/watch?v=999q3_htKPU)
```

Comment:
Looks weird. I would link all of the beginning, and then link "Part 2" with the second video